ros/angel_system_nodes/angel_system_nodes/task_monitor.py

```diff
@@ -145,6 +148,14 @@ def __init__(self):
 
         self._task = CoffeeDemoTask(self._task_steps)
 
+        self._task_graph_service = self.create_service(
+            QueryTaskGraph,
+            "query_task_graph",
+            self.query_task_graph_callback
+        )
+
+        self._task = CoffeeDemoTask()
```

Comment:
Small error in the rebase: this line was modified and master, which is present on L149 above, so remove this line.